invoices/invoiceregistry.go

```diff
@@ -12,6 +13,10 @@ import (
 	"github.com/lightningnetwork/lnd/queue"
 )
 
+const (
+	defaultInvoiceExpiry = 3600 * time.Second
```

Comment:
Missing a `godoc` comment.

Reply:
This constant is already defined in `zpay32.Invoice.Expiry()`

Reply:
Moved constant to `channeldb/invoices.go` instead as the code has changed during the fixup. IMHO it's not worth using `zpay32.Invoice.Expiry()` here as that'd require creating a zpay32.Invoice.

Reply:
You can extract `3600 * time.Second` into an exported symbol in `zpay32`